server/app/views/components/FieldWithLabel.java

```diff
@@ -164,6 +165,11 @@ public FieldWithLabel setPlaceholderText(String placeholder) {
     return this;
   }
 
+  public FieldWithLabel setAutocomplete(Optional<String> autocomplete) {
```

Comment:
nit: Can we add a comment? Something like:

"
Sets the autocomplete attribute. 
The provided string must come from the predefined list here: https://www.w3.org/TR/WCAG21/#input-purposes.
"

Reply:
Good call. Done!